### PR TITLE
sentiment list extension

### LINKED ISSUE #47: [Twitter] Classify remaining characters
Currently the crawler shows a list of "unclassified" characters at every start:

```
[crawler/blacklist][WARN] filtered unclassified: Willam
[crawler/blacklist][WARN] filtered unclassified: Willit
[crawler/blacklist][WARN] filtered unclassified: Willum
[crawler/blacklist][WARN] filtered unclassified: Wolmer
[crawler/blacklist][WARN] filtered unclassified: Wulfe
[crawler/blacklist][WARN] filtered unclassified: Wylla
...
```

Those are characters with names that are rather short and could possibly be "unsearchable" on Twitter.

Those remaining characters have to be classified into the whitelist and the blacklist (config.json[.sample]) like this:
1. Go to `https://twitter.com/search?f=tweets&vertical=default&q="{charactername}"&src=sprv`
2. If there are either not many tweets at all or at least half of the posts are about the GoT character, add it the the whitelist, otherwise add it to the blacklist.

### PR DESCRIPTION
**Pull request by ghost**
sentiment word list added

Fixes #47

### CODE REVIEW

File: defaults.json [hunk 1]
```diff
@@ -296,13 +619,88 @@
...
+        "fck": -4,
+        "fck u": -4,
+        "coolest": 2,
+        "lord": 3,
+        "princess": 3,
+        "boss": 3,
+        "humble": 2,
+        "smug": -2,
+        "feast": 2,
+        "woo hoo": 3,
```
Comment (julienschmidt) [inline defaults.json]: we use a tokenizer which splits at every whitespace. So this world could never be matched.

File: defaults.json [hunk 2]
```diff
@@ -296,13 +619,88 @@
...
+        "seize": -2,
+        "spoil": -3,
+        "spoiler": -3,
+        "epic": 2,
+        "my lord": 2,
+        "puke": -2,
+        "pukes": -2,
+        "fruitless": -2,
+        "offence": -2,
+        "RIP": -3,
```
Comment (julienschmidt) [inline defaults.json]: RIP is negative?

File: defaults.json [hunk 3]
```diff
@@ -296,13 +619,88 @@
...
+        "present": 2,
+        "fairy": 2,
+        "fairy tale": 2,
+        "lengthy": -1,
+        "finer": 2,
+        "fought": -3,
+        "power": 2,
+        "seize": -2,
+        "spoil": -3,
+        "spoiler": -3,
```
Comment (julienschmidt) [inline defaults.json]: spoiler has nothing to do with the sentiment abou the character

Comment (ghost) [inline defaults.json]: removing spoiler

File: defaults.json [hunk 4]
```diff
@@ -296,13 +619,88 @@
...
+        "junk": -2,
+        "independent": 2,
+        "beast": -3,
+        "cutest": 2,
+        "birth": 1,
+        "birthday": 2,
+        "bald": -1,
+        "against": -2,
+        "stand": 2,
+        "gracias": 2,
```
Comment (sacdallago) [inline defaults.json:662]: Ouh, if it wasn't clear: gracias means "thank you" and that guy is surely not expressing neither happy or unhappy feelings! :laughing: 
I honestly don't think "thank you" has any information on + or - feelings, but **I might be wrong**

File: defaults.json [hunk 5]
```diff
@@ -296,13 +619,88 @@
...
+        "smug": -2,
+        "feast": 2,
+        "woo hoo": 3,
+        "junk": -2,
+        "independent": 2,
+        "beast": -3,
+        "cutest": 2,
+        "birth": 1,
+        "birthday": 2,
+        "bald": -1,
```
Comment (julienschmidt) [inline defaults.json]: ???

File: defaults.json (hunk 2 above)
Comment (ghost) [inline defaults.json]: I am confused with this word... @marcusnovotny any view, what should be the sentiment of "RIP"

File: defaults.json (hunk 1 above)
Comment (ghost) [inline defaults.json]: removing "woo hoo"

File: defaults.json (hunk 5 above)
Comment (sacdallago) [inline defaults.json]: Not cool @santanumohanta .
There are _some people_ on the dark path of baldness around here. Not cool.

:joy:

Comment (ghost) [inline defaults.json]: removing bald , ya it does not make sense .

File: defaults.json (hunk 4 above)
Comment (ghost) [inline defaults.json:662]: in Afinn-111 wordlist I see thank/thanks has +2 valance

File: defaults.json (hunk 1 above)
Comment (julienschmidt) [inline defaults.json]: This applies to all words containing spaces. You could just leave `woo` here.

**Comment by sacdallago** [comment]
In your last commit you are still also including space-separated words. @julienschmidt already said that those don't count, please remove them all if they don't singularly contribute

File: defaults.json (hunk 1 above)
Comment (ghost) [inline defaults.json]: okay applying for others as well . woo already exists in afinn-111 list

File: defaults.json (hunk 2 above)
Comment (marcusnovotny) [inline defaults.json]: Sorry, missed this one. I'd agree with Julien, seems more positive to me. It's basically paying your respects :smile:

**Comment by marcusnovotny** [comment]
Sorry for not joining in earlier. Only two things two add:
- Why has birthday a positive score?
- tough can also be a compliment, as in "Daenerys is a tough girl". I think it might be used quite often in that context

File: defaults.json (hunk 2 above)
Comment (ghost) [inline defaults.json]: giving it positive valance 2
"RIP": 2

Comment (ghost) [inline defaults.json]: @marcusnovotny - I agree with you "tough" can also be used in positive way when it comes to a character, so it seems it's better to give it a positive valance(+2)! 
and for birthday , it is always associated with some occasion like for example... "lets have a get together for Jon's birthday". no negative scenario is coming in my mind. So I think it should be positive

Comment (marcusnovotny) [inline defaults.json]: Apply your last changes & then merge :)

Comment (julienschmidt) [inline defaults.json]: @santanumohanta we are still waiting for your fixes...

File: defaults.json [hunk 6]
```diff
@@ -296,13 +619,81 @@
...
+        "Yorkel",
+        "Ysilla",
+        "Zei",
+        "Zekko",
+        "Zollo"
+    ],
+
+    "sentimentlist": {
+        "honorable": 2,
+        "nothing": -1,
```
Comment (julienschmidt) [inline defaults.json]: nothing is also pretty.. neutral

**Pull request merged**